tests/atomistics/job/test_atomistic.py

```diff
@@ -77,19 +74,16 @@ def test_get_structure(self):
         # tested here with lammps as a concrete instantiation of AtomisticGenericJob
         # have to do extra tango because Project.unpack is weird right now
         cwd = os.curdir
-        to_dir = os.path.join(self.project.path, "..")
-        shutil.copy("tests/static/lammps_test_files/get_structure_test.tar.gz", to_dir)
-        shutil.copy("tests/static/lammps_test_files/export.csv", to_dir)
-        os.chdir(to_dir)
+        tests_loc = Path(__file__).parents[2]
+        shutil.copy(os.path.join(tests_loc, "static/lammps_test_files/get_structure_test.tar.gz"), cwd)
+        shutil.copy(os.path.join(tests_loc, "static/lammps_test_files/export.csv"), cwd)
         self.project.unpack("get_structure_test")
         job = self.project.load("inter_calculator")
-        os.chdir(cwd)
+        os.unlink("export.csv")
+        os.unlink("get_structure_test.tar.gz")
 
         for i, struct in enumerate(job.iter_structures()):
             # breakpoint()
             self.assertTrue(np.allclose(job.output.positions[i], struct.positions))
             self.assertTrue(np.allclose(job.output.cells[i], struct.cell.array))
             self.assertTrue(np.allclose(job.output.indices[i], struct.indices))
-
-if __name__ == "__main__":
```

Comment:
Ah, sorry, I didn't see this in the mobile app.

Anyhow, it really necessary? I guess if you do `python tests/atomistics/job/test_atomistic.py` it is, but if you do `python -m unittest discover tests/atomistics/job` it will run this and the other ones there... but maybe you can append `/test_atomistic.py` to get *just* these? Tbh I haven't played around with it much from command line, but I'm 99% sure you can swing it this way

Reply:
Yes, with `python -m unittest path/to/test_file.py` you run all tests in this file (If you point to one specific one, you do not need to discover). So Sam, just make an `alias run_test_py="python -m unittest discover "` and you may fire any directory you want by `run_test_py test_dir` :)